docs/reference/release-notes/850.md

```diff
@@ -0,0 +1,188 @@
+---
+id: 850
+title: "8.5 Release notes"
+description: "Release notes for 8.5, including alphas"
+---
+
+## Enhancements
+
+### Role Support for Web Modeler with OIDC in SM
+
+You can now use Role-Based Access Control with your own OIDC Identity Provider (i.e. Entra ID) and the Web Modeler without relying on Keycloak. This extends RBAC and Role Mapping support that is available for other components to Web Modeler.
+
+### Introductory UI header for processes page
+
+The process overview page was redesigned for clarity and ease-of-use.
+
+### Fully supported Spring Zeebe SDK
+
+Use supported Spring Zeebe SDK in your Spring or Spring Boot projects to interact with Camunda 8 engine API and build process applications. Spring Zeebe SDK works with Zeebe gRPC and the new Zeebe REST API.
```

Comment:
What do we mean by "Camunda 8 engine API"?

Reply:
I _think_ this is a reference to the core components vs. Web Modeler, for example. I will follow up.

Reply:
Follow up here - https://camunda.slack.com/archives/C02AWA0RF8A/p1713198528757339

Reply:
By `Camunda 8 engine API` I meant Zeebe API - we can change it to Zeebe to me it easier to understand

Reply:
Will do!

Reply:
https://github.com/camunda/camunda-docs/pull/3696